### DISCUSSION TITLE
CouchDB Update

### DISCUSSION THREAD

**Discussion by knario** [community-scripts/ProxmoxVE#579] [Q&A]
Good morning! I have installed couchDB using the script. 
My question is how can I update CouchDB, because the update command shows me that there is no CouchDB installed on the system.
I'm not in a hurry, because it works very well to synchronize obsidian. It's just a doubt.

Happy weekend to all the wonderful people who make up this community left by Tteck !!!

**Comment by MickLesk** [comment]
Currently we dont have an Update in the Script. 

Can you check this & give Feedback?
https://www.gevernova.com/software/documentation/proficy-plant-applications/version2022/t_pa_webclient_upgrading_apache_couchdb_recent_version.html

**Reply by knario** [reply]
Thank you very much for your response. 
I'll try it tomorrow and see if it works.

**Reply by knario** [reply]
Hi MickLesk:
My mistake, I think using apt update && apt upgrade updates to the latest version.
I checked and right now I have couchDB version 3.4.2, which is the latest. 
Sorry I didn't check before.  😅
